src/app/admintoolbar/admintoolbar.component.html

```diff
@@ -0,0 +1,16 @@
+<nav class="navbar navbar-inverse">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="#" style="color:rgb(236, 160, 60); font-size: 40px">Number</a>
```

Comment:
Try to move all styles to the .css file of the component. Preferably keep it under a class to that it is easier to re-use later.
Take care of it in the lines 4, 5, 8, 9, 12.

Reply:
Kept them separately to make the code readable. Also, since different color scheme are used it would make more sense to write the styles separately. Let me know if you still want changes.

Reply:
After integration, we will have 3 dashboards and toolbars which will use similar styles and themes. It's better to keep them at a common css location for all of them to access it.